README.markdown

```diff
@@ -314,32 +314,36 @@ from the XML retrieved from the MedLine website (see doc/medline for more detail
 
     rake usasearch:medline:load
 
-## Affiliate accounts
-The database is seeded with a user that has super admin privileges:
-    
-    email: admin@email.gov
-    password: test1234!
-
-You can also create a user account using a bogus .gov or .mil email address:
-
-<http://127.0.0.1:3000/login>
-
-Look for the `email_verification_token` in your rails server stdout and open the verification link in your favorite browser:
-    
-    http://127.0.0.1:3000/email_verification/<email_verification_token>
+## Creating a new local admin account
+[Login.gov](https://login.gov) is used for authentication.
 
-Create an affiliate for yourself called 'foo', and put in a simple header/footer like H1's or something.
-Re-run your 'taxes' search and add '&affiliate=foo' to the HTTP request.
+To create a new local admin account we will need to:
+1. Create an account on Login's sandbox environment.
+2. Get the Login sandbox private key from a team member.
+3. Add an admin user to your local app.
 
-## Analytics
-Give your user account admin privileges. Here's how with rails console:
+#### 1. Login sandbox
+[Create an account](https://idp.int.identitysandbox.gov/sign_up/enter_email) on Login's sandbox environment. This will need to be a real, valid government email address that you can get emails at. Something like `your-real-name+search-local@gsa.gov`. You'll receive a validation email to click then set a password. You will also need to set a second auth method, such as your PIV card.
```

Comment:
Let's simplify this as `set a password and secondary authentication method.` Not everyone using this repo will know what a PIV card is (*cough* me *cough*), so we can avoid confusion and just let people follow the instructions on Login.gov.